roles/kubernetes-apps/ansible/templates/calico-policy-controller.yml.j2

```diff
@@ -2,7 +2,6 @@ apiVersion: extensions/v1beta1
 kind: ReplicaSet
 metadata:
   name: calico-policy-controller
-  namespace: kube-system
```

Comment:
if user changes it manually then applies w/o a namespace given, he'd be in trouble. Let's paremetrize instead and specify in both places